strict/src/Data/Strict/Maybe.hs

```diff
@@ -67,7 +81,17 @@ import           Data.Data           (Data (..), Typeable1 (..))
 #endif
 
 #if __GLASGOW_HASKELL__ >= 706
-import           GHC.Generics        (Generic (..))
+import           GHC.Generics        (Generic, Generic1)
+#endif
+
+#if MIN_VERSION_deepseq(1,4,3)
+import Control.DeepSeq (NFData1 (..))
+#endif
+
+#ifdef LIFTED_FUNCTOR_CLASSES
+import Data.Functor.Classes (Eq1 (..), Ord1 (..), Read1 (..), Show1 (..))
+#else
+import Data.Functor.Classes (Eq1 (..), Ord1 (..), Read1 (..), Show1 (..))
```

Comment:
Both branches here are identical, or is there a reason to do this on purpose (future extensions)?

Reply:
Yeah, I'd like to keep structure ~similar.